### PR TITLE
Withholds love from the dead.

### LINKED ISSUE #15509: Valentine cards spawn on every human mob, alive or dead
This leads to situations like away missions littered with corpses now also giving every spawned corpse their own valentine card.

I don't know if this makes the missions more morbid or not.

### PR DESCRIPTION
**Pull request by harmonyn**
- Valentines and candy hearts will only be distributed to the living during Valentine's Day events. Would fix https://github.com/tgstation/tgstation/issues/15509 sorry cheridan
- Adds a few new messages.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
:(

**Comment by bgobandit** [comment]
AGH WHAT THE HELL

**Comment by pudl** [comment]
It's a feature dingus

**Comment by bgobandit** [comment]
the original PR raised a good point that it can be used to metagame corpse locations

**Comment by Cheridan** [comment]
;__;

**Comment by Crushtoe** [comment]
The dead deserve love.

File: code/modules/events/holiday/vday.dm [hunk 1]
```diff
@@ -73,7 +72,9 @@
 	               "If you had a pickaxe you'd be a shaft FINEr.",
 	               "Roses are red, tide is gray, if I were an assistant I'd steal you away.",
 	               "Roses are red, text is green, I love you more than cleanbots clean.",
-	               "If you were a carp I'd fi-lay you." )
+	               "If you were a carp I'd fi-lay you.",
+	               "I'm a nuke op, and my pinpointer leads to your heart.",
+	               "Wanna slay my megafauna?" )
```
Comment (JJRcop) [inline code/modules/events/holiday/vday.dm]: .

Comment (JJRcop) [inline code/modules/events/holiday/vday.dm]: .

**Pull request merged**